src/usr/lib/rsetup/tui/hardware/hardware.sh

```diff
@@ -386,11 +386,12 @@ __check_service_status() {
 __hardware_otg() {
     checklist_init
 
-    local udc udc_function udc_function_list status i
+    local udc udc_function udc_function_list services status i
+    readarray -t services < <(jq -r '.[]' /usr/share/radxa-otgutils/services.json)
```

Comment:
```suggestion
    readarray -t services < <(jq -er '.[]' /usr/share/radxa-otgutils/services.json)
```
然后要加一个错误处理，如果这个文件不存在则默认是以前的服务列表